realm/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -177,10 +199,13 @@ public double getDouble(String fieldName) {
 
     /**
      * Returns the {@code byte[]} value for a given field.
+     * If the field is nullable use {@link #isNull(String)} to check for {@code null} instead of using
```

Comment:
Can't we do like we do for `Date`? That is, return `null` if the value is `null`?